app/forms/conversion/create_project_form.rb

```diff
@@ -14,6 +14,7 @@ class NegativeValueError < StandardError; end
   attribute :advisory_board_conditions
   attribute :note_body
   attribute :user
+  attribute :assigned_to_regional_caseworker_team, :boolean
```

Comment:
I think we can declare this in the `Conversion::Voluntary::CreateProjectForm` as it is only for that class of project?